mods/bones/init.lua

```diff
@@ -155,21 +161,24 @@ local function may_replace(pos, player)
 	return node_definition.buildable_to and not minetest.is_protected(pos, player:get_player_name())
 end
 
+if minetest.setting_getbool("creative_mode") then
+	return
```

Comment:
Wouldn't it be better to explicitly disable bones in creative mode with the setting, so players can re-enable it again if they want to?